### PR TITLE
support mjpeg stream screenshot via new mjpegScreenshotUrl cap

### PR DESCRIPTION
**Pull request by jlipps**
Implementation for the uiautomator2 driver of https://github.com/appium/appium-base-driver/pull/230, which this depends on.

### CODE REVIEW

**Comment by jlipps** [comment]
Haven't gotten to tests for this yet; probably need to move the test mjpeg server to appium-test-support so we can use it in multiple places.

File: lib/commands/screenshot.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
+import logger from '../logger';
+
 let commands = {};
 
 commands.getScreenshot = async function () {
+  if (this.mjpegStream) {
+    const data = this.mjpegStream.lastChunkBase64;
+    if (data) {
+      return data;
```
Comment (mykola-mokhnach) [inline lib/commands/screenshot.js]: I still don't think that it is a good idea to break the spec this way and send images in jpeg format instead of png

Comment (mykola-mokhnach) [reply lib/commands/screenshot.js]: we could probably use jimp for this purpose

**Comment by jlipps** [comment]
ok, PR updated to return PNG data and to include tests

**Comment by jlipps** [comment]
package.json updated, let's see about CI

**Comment by jlipps** [comment]
CI is not happy, however it's some weird error about node version and buffertools native module not compiling.

**Comment by jlipps** [comment]
ah, @imurchie found the issue.

**Comment by jlipps** [comment]
pushed @imurchie's android build fixes in this PR as well

**Comment by jlipps** [comment]
ok, build passed.

**Comment by jlipps** [comment]
published as `1.15.0`

**Pull request merged**